requirements/requirements_tts.txt

```diff
@@ -1,6 +1,7 @@
 attrdict
 einops
 inflect
+kornia
```

Comment:
we need tests to verify that this does not affect other dependencies and break any test cases.

Reply:
How would one do that? CI says tests pass

It is used for a single op though, could be rewritten in PyTorch